Tryouts/Prototypes/ModulesPrototype/Program.cs

```diff
@@ -135,28 +134,27 @@ public static async Task Main(string[] args)
         {
             //for the demo's sake we are starting just the Process Explorer.
             //the other applications, that are declared in the manifest can be started via the Process Explorer frontend
-            if (module.Value.Name != "processExplorerService") continue;
+            //if (module.Value.Name != "processExplorerService") continue;
             module.Value.State = ModuleState.Started;
             loader.RequestStartProcess(new LaunchRequest { name = module.Value.Name, instanceId = module.Key });
-            Thread.Sleep(5000);
         }
 
-        if (!instances
-            .Where(module =>
-                module.Value.Name == "processExplorerService"
-                && (module.Value.State == SubsystemState.Started || module.Value.State == SubsystemState.Running))
-            .Any())
-            goto endState;
-
-        await infoCollector.InitializeSubsystemControllerRouteAsync();
-        var consoleShellPrototype = new ProcessInformation(Process.GetCurrentProcess());
-        processInfo.Add(consoleShellPrototype);
-        var serializedInstances = JsonSerializer.Serialize(instances);
-        if (serializedInstances != string.Empty) await infoCollector.SendRegisteredSubsystemsAsync(serializedInstances);
-        if (processInfo.Count != 0) await infoCollector.SendInitProcessInfoAsync(processInfo);
-        await infoCollector.EnableProcessMonitorAsync();
-
-    endState:
+        //if (!instances
```

Comment:
What's the plan for these commented out lines?

Reply:
TBH that was just a quick fix to get back to the original state where the modules are loaded and started instead of Process Explorer. IMO this code should never have been merged, and Process Explorer should have its own demo app.

Reply:
Tagging @lilla28 - can you please clarify what would be the ideal course of action here?